01-JS-DOM-APIs/js/First.js

```diff
@@ -0,0 +1,65 @@
+function messageFirst() {
+  setTimeout(function () {
+    document.getElementById("content").innerHTML = "Hello world";
+  }, 2000);
+}
+
+function buttonClick(){
+  let config = {
+    url: 'http://api.icndb.com/jokes/random'
+  }
+  function resolve(xhttp) {
+    var response = JSON.parse(event.target.response);
+    document.getElementById('joke').innerHTML = response.value.joke;
+  }
+  function reject(){
+
+  }
+  eventRequest(config).then(resolve, reject);
+}
+
+function git(){
+  let searchField = document.getElementById("searchField").value;
+  console.log(searchField);
```

Comment:
It is okey to use `console.log` in order to debug and develop. But remember to delete before commit code.